GooglePay/src/main/java/com/braintreepayments/api/GooglePayReadinessResult.kt

```diff
@@ -0,0 +1,19 @@
+package com.braintreepayments.api
+
+/**
+ * Result of [GooglePayClient.isReadyToPay]
+ */
+sealed class GooglePayReadinessResult {
```

Comment:
Still open to other ideas on the naming of this result object

Reply:
Looking through the Google docs they seem to use "readiness" as well so I think that's fine. 🤷

Reply:
Yeah I think that's great- we align with Google's naming conventions so it's easier for devs to understand the integration.